onnx/defs/nn/defs.cc

```diff
@@ -345,8 +345,8 @@ namespace onnx {
             schema.NumInputs(1);
             schema.NumOutputs(1);
             schema.Attr("p",
-                        "p value of the Lp norm used to pool over the input data, default is 2.0.",
-                        AttrType::FLOAT);
+                        "p value of the Lp norm used to pool over the input data, default is 2.",
+                        AttrType::INT);
```

Comment:
breaking change

Reply:
I will change it. @bddppq so why did it pass all tests?

Reply:
Done.

Reply:
The test coverage is pretty low in onnx repo :(
Besides, "ONNX Backend Test" (i.e. those test cases in onnx/backend/test/) is not exercised in onnx repo since it needs a backend (onnx-caffe2, onnx-tensorflow ...) to run.